tarodb/assets_store_test.go

```diff
@@ -235,17 +235,8 @@ func randAsset(t *testing.T, genOpts ...assetGenOpt) *asset.Asset {
 	return newAsset
 }
 
-func assetWitnessEqual(t *testing.T, a, b []asset.Witness) {
-	t.Helper()
-
-	require.Equal(t, len(a), len(b))
-
-	for i := 0; i < len(a); i++ {
-		witA := a[i]
-		witB := b[i]
-
-		require.True(t, witA.DeepEqual(&witB))
-	}
+func assertAssetEqual(t *testing.T, a, b *asset.Asset) {
+	require.True(t, a.DeepEqual(b))
```

Comment:
Okay, I see why the more verbose equality check was nice. Now it's very hard to find out why the assets aren't equal.
This is the "compromise" I used to locate the current flake:
```
	equal := a.DeepEqual(b)
	if !equal {
		// Print a nice diff if the native equality check fails.
		require.Equal(t, b, a)
		
		// Make sure we fail in any case, even if the above equality
		// check succeeds (which shouldn't be the case).
		t.Fatalf("Asset equality failed!")
	}
```

Reply:
I think this is fixed by removing this line:

https://github.com/lightninglabs/taro/blob/581331708fa32e09065884399fbad18309128088/tarodb/assets_store_test.go#L323